src/plugins/data/common/es_query/index.ts

```diff
@@ -18,6 +18,7 @@
  */
 import * as esQuery from './es_query';
 import * as esFilters from './filters';
-import * as utils from './utils';
+import * as esKuery from './kuery';
+import * as esUtils from './utils';
```

Comment:
I'm not sure that we should rename this to `esUtils`.
I don't think this is going to be the correct prefix.
I would keep it as is (`utils`).

Reply:
np, let's keep it as it